run.go

```diff
@@ -57,6 +57,10 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 			Name:  "no-new-keyring",
 			Usage: "do not create a new session keyring for the container.  This will cause the container to inherit the calling processes session key",
 		},
+		cli.IntFlag{
+			Name:  "preserve-fds",
+			Usage: "Pass N additional file descriptors to the container (stdio + $LISTEN_FDS + N in total)",
```

Comment:
That's a good point. I was a bit surprised that `IntSliceFlag` didn't support comma separated lists or even `N-M` spans (or if it does that the syntax wasn't more obvious...).

FWIW I did confirm that `Cmd.ExtraFiles` looks on first glance to DTRT with `nil` in the list, although `finalizeNamespace` and `CloseExecFrom` would probably need extending to handle a sparse fd space to be absolutely safe.